playbooks/luna.yml

```diff
@@ -0,0 +1,20 @@
+---
+- hosts: all
+  become: true
+  vars:
+    foreman_installer_options:
+      - "--disable-system-checks"
+      - "--foreman-admin-password {{ foreman_installer_admin_password }}"
+      - "--enable-foreman-plugin-ansible"
+      - "--enable-foreman-plugin-discovery"
+      - "--enable-foreman-plugin-bootdisk"
+      - "--enable-foreman-plugin-hooks"
+      - "--enable-foreman-plugin-openscap"
+      - "--enable-foreman-plugin-templates"
+      - "--enable-foreman-plugin-remote-execution"
+      - "--enable-foreman-proxy-plugin-remote-execution-ssh"
+      - "--enable-foreman-proxy-plugin-discovery"
+      - "--enable-foreman-proxy-plugin-ansible"
+      - "--enable-foreman-proxy-plugin-openscap"
```

Comment:
I may be missing something, but doesn't it need extra records in https://github.com/theforeman/forklift/blob/master/roles/foreman_installer_devel_scenario/files/katello-devel-answers.yaml#L32 ?

Reply:
I think you are right. Surprised the installer does not complain.

Reply:
Oh wait, no, this uses the standard Katello scenario. Note this is not the same as luna-devel, this does a nightly production install with all of these enabled.

Reply:
Could we get a nightly production luna box added to the `boxes.d/99-local.yaml.example`, or (bonus points) enable luna box configuration with `vagrant/config/versions.yaml`.